.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Remsfal Backend
+
+Remsfal Backend is an open-source backend service for managing and serving data for the Remsfal project. This project is part of an university curriculum and is being partly developed by a group of students.
+
+## Building and Working with the Codebase
+
+For instructions on how to build and work with the Remsfal backend codebase, please refer to the [Building Guide](https://github.com/remsfal/remsfal-backend/blob/main/README.md).
```

Comment:
The flies placed in the ".github" repository are [default community health files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#creating-a-repository-for-default-files). They are displayed in all remstal repositories. So could you please include the frontend too?